src/components/accounts/accounts.js

```diff
@@ -82,6 +82,15 @@ export class AccountsComponent {
     });
   }
 
+  updateAccountDataDelete(transaction) {
+    AccountListService.update(
+      transaction.account,
+      -parseInt(transaction.type + transaction.amount)
```

Comment:
please explain this lines -- what is `transaction.type` ?

Reply:
it is income or expence